webknossos/webknossos/skeleton/skeleton.py

```diff
@@ -104,6 +105,43 @@ def save(self, out_path: Union[str, PathLike]) -> None:
         annotation = Annotation(name=out_path.stem, skeleton=self, time=None)
         annotation.save(out_path)
 
+    def add_nx_graphs(
+        self, tree_dict: Union[List[nx.Graph], Dict[str, List[nx.Graph]]]
+    ) -> None:
+        """
+        A utility to add nx graphs [NetworkX graph object](https://networkx.org/) to a wk skeleton object. Accepts both a simple list of multiple skeletons/trees or a dictionary grouping skeleton inputs.
+
+        Arguments:
+        tree_dict (Union[List[nx.Graph], Dict[str, List[nx.Graph]]]): A list of wK tree-like structures as NetworkX graphs or a dictionary of group names and same lists of NetworkX tree objects.
+        """
+
+        if not isinstance(tree_dict, dict):
+            tree_dict = {"main_group": tree_dict}
```

Comment:
What was the behavior of the old `generate_nml`? Did it name the root group "main_group", too?

Reply:
Yes

Reply:
Yes